specification/migrate/create_from/MigrateCreateFromRequest.ts

```diff
@@ -23,8 +23,9 @@ import { IndexName } from '@_types/common'
 import { TypeMapping } from '@_types/mapping/TypeMapping'
 
 /**
- * This API creates a destination from a source index. It copies the mappings and settings from the source index while allowing request settings and mappings to override the source values.
+ * Create a destination from a source index.
```

Comment:
Maybe `Create an index from a source index`?

Reply:
Thanks! Addressed in a1f8330f5fe05776ae8877c187104fe7920f79ef